test/e2e/autotls/config/util.go

```diff
@@ -67,13 +67,17 @@ func DeleteDNSRecord(record *DNSRecord, svcAccountKeyFile, dnsProject, dnsZone s
 
 // ChangeDNSRecord changes the given DNS record.
 func ChangeDNSRecord(change *dns.Change, svc *dns.Service, dnsProject, dnsZone string) error {
+	chg, err := svc.Changes.Create(dnsProject, dnsZone, change).Do()
+	if err != nil {
```

Comment:
can you get `already exists` here?

Reply:
The domain name of the DNS record for each test round is kind of unique as it has the format of {random number}.{project id}.{domain}. So it is unlikely to get AlreadyExists error when firstly creating the DNS record.